lib/check/check.py

```diff
@@ -173,12 +173,15 @@ def import_check(check_path: str) -> ModuleType:
     return lib
 
 
-def set_output_options(quiet: bool, output_modes: list, input_output_directory: str):
+def set_output_options(
+    quiet: bool, output_modes: list, input_output_directory: str, security_hub: bool
```

Comment:
If `security_hub` is a boolean value I prefer something like `security_hub_enabled` or `output_to_security_hub` is more clear. What do you think?

Reply:
Yes, I like more your suggestion.